[
  {
    TraceId: Id_1,
    SpanId: Id_2,
    Name: CustomTracingExceptionHandler.handle-async,
    Resource: CustomTracingExceptionHandler.handle-async,
    Service: sample/my-app,
    ParentId: Id_3,
    Tags: {
      deployment.environment: integration_tests,
      language: dotnet,
      span.kind: server,
      version: 1.0.0
    }
  },
  {
    TraceId: Id_1,
    SpanId: Id_3,
    Name: GET,
    LogicScope: aspnet.request,
    Resource: GET /api2/transientfailure/false,
    Service: sample/my-app,
    Type: web,
    Error: 1,
    Tags: {
      deployment.environment: integration_tests,
      http.method: GET,
      http.request.headers.host: localhost:00000,
      http.status_code: 500,
      http.url: http://localhost:00000/my-app/api2/transientfailure/false,
      language: dotnet,
      net.peer.ip: ::1,
      runtime-id: Guid_1,
      sfx.error.kind: System.ArgumentException,
      sfx.error.message: Passed in value was not 'true': false,
      sfx.error.stack:
System.ArgumentException: Passed in value was not 'true': false
at Samples.AspNetMvc5.Controllers.ConventionsController.TransientFailure(String value),
      signalfx.tracing.library: dotnet-tracing,
      signalfx.tracing.version: x.y.z,
      span.kind: server,
      version: 1.0.0
    },
    Metrics: {
      process_id: 0,
      _dd.top_level: 1.0,
      _dd.tracer_kr: 1.0,
      _sampling_priority_v1: 1.0
    }
  }
]